### PR TITLE
fix(clients): mention `main` branch instead of `master`

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: -

### Changes included:

closes https://github.com/algolia/algoliasearch-client-csharp/issues/858, also fixes the Go client that targets the master branch

### CODE REVIEW

**Review by millotp — COMMENTED** [review]
can you fix go in README.md pls too ? and kotlin README.md

**Comment by shortcuts** [comment]
> can you fix go in README.md pls too ? and kotlin README.md

kotlin requires a bit more work looks like it's totally outdated

**Review by millotp — APPROVED** [review]
okay too bad

**Comment by shortcuts** [comment]
> okay too bad

actually I'll do it and STOP BEING LAZY brb

**Comment by shortcuts** [comment]
I've also updated the new links to the doc

File: templates/kotlin/README.mustache [hunk 1]
```diff
@@ -114,24 +114,16 @@ var response = client.search(
...
 ## Contributing
 
-This repository hosts the code of the generated Algolia API client for Kotlin, if you'd like to contribute, head over to the [main repository](https://github.com/algolia/api-clients-automation). You can also find contributing guides on [our documentation website](https://api-clients-automation.netlify.app/docs/introduction).
+This repository hosts the code of the generated Algolia API client for Go, if you'd like to contribute, head over to the [main repository](https://github.com/algolia/api-clients-automation). You can also find contributing guides on [our documentation website](https://api-clients-automation.netlify.app/docs/introduction).
 
 ## 📄 License
 
-The Algolia Kotlin API Client is an open-sourced software licensed under the [MIT license](LICENSE).
\ No newline at end of file
+The Algolia Go API Client is an open-sourced software licensed under the [MIT license](LICENSE).
```
Comment (millotp) [inline templates/kotlin/README.mustache]: this shouldn't change

**Review by millotp — COMMENTED** [review]
huge !

File: templates/kotlin/README.mustache (hunk 1 above)
Comment (shortcuts) [reply templates/kotlin/README.mustache]: Ah oops ahah

File: templates/kotlin/README.mustache [hunk 2]
```diff
@@ -114,19 +114,11 @@ var response = client.search(
...
-- [ExceptionHandling](https://github.com/algolia/algoliasearch-client-kotlin/tree/master/docs/ExceptionHandling.md)
-- [Configure the HTTP client](https://github.com/algolia/algoliasearch-client-kotlin/tree/master/docs/HTTPClient.md)
+For full documentation, visit the **[Algolia Kotlin API Client](https://www.algolia.com/doc/libraries/kotlin/)**.
 
 ## ❓ Troubleshooting
 
-Encountering an issue? Before reaching out to support, we recommend heading to our [FAQ](https://www.algolia.com/doc/api-client/troubleshooting/faq/kotlin/) where you will find answers for the most common issues and gotchas with the client. You can also open [a GitHub issue](https://github.com/algolia/api-clients-automation/issues/new?assignees=&labels=&projects=&template=Bug_report.md)
+Encountering an issue? Before reaching out to support, we recommend heading to our [FAQ](https://www.algolia.com/doc/api-client/troubleshooting/faq/go/) where you will find answers for the most common issues and gotchas with the client. You can also open [a GitHub issue](https://github.com/algolia/api-clients-automation/issues/new?assignees=&labels=&projects=&template=Bug_report.md)
```
Comment (millotp) [inline templates/kotlin/README.mustache:121]: this too

**Review by millotp — APPROVED** [review]

**Pull request merged**